actions/qemuimg2disk/v1/partprobe.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"golang.org/x/sys/unix"
+	"os"
+)
+
+func doit() error {
+	disk := os.Getenv("DEST_DISK")
+	fileOut, err := os.OpenFile(disk, os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return err
+	}
+	defer fileOut.Close()
+
+	// Do the equivalent of partprobe on the device
+	if err := fileOut.Sync(); err != nil {
+		return fmt.Errorf("failed to sync the block device")
+	}
+
+	if err := unix.IoctlSetInt(int(fileOut.Fd()), unix.BLKRRPART, 0); err != nil {
+		return fmt.Errorf("error re-probing the partitions for the specified device: %v", err)
+	}
+
+	return nil
+}
+
+func main() {
+	doit()
```

Comment:
I do love a `doit()` function.. but is it needed given everything could live in `main()` :-)

Reply:
The only reason why a function like this is useful is for testing. But there are not unit tests for it (also I think it is easier for testing if you get the env var in main and pass it to do function.  Overall I agree with Dan, as it is written it can leave in main directly

Reply:
Copypasta remnants from copying code from elsewhere in the repo.